browser/ui/webui/settings/brave_vpn/brave_vpn_handler.h

```diff
@@ -26,7 +26,7 @@ class BraveVpnHandler : public settings::SettingsPageUIHandler,
   void RegisterMessages() override;
 
   void HandleRegisterWireguardService(const base::Value::List& args);
```

Comment:
if we're using `Installed` it should be used consistently

Reply:
Fixed in https://github.com/brave/brave-core/pull/20754/commits/bf5eb72f94cbd0b3386babee6b2c0b43434783ea